src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1350,6 +1370,16 @@ public static int setCredentialsCacheMaxEntries(int maxEntries)
         return conf.credentials_cache_max_entries = maxEntries;
     }
 
+    public static boolean getCredentialsCacheActiveUpdate()
+    {
+        return conf.credentials_cache_active_update;
+    }
+
+    public static void setCredentialsCacheActiveUpdate(boolean activeUpdate)
```

Comment:
nitty nit: We call this `activeUpdate` here and `update` in the other two setters.

Reply:
I think "nitty nit" is my new favorite term.